Gemfile.lock

```diff
@@ -247,3 +247,6 @@ DEPENDENCIES
   simplecov
   timecop
   yard
+
```

Comment:
It would be good to revert this bit and upgrade to bundler 1.10.5, which stops doing this silly thing.